scipy/optimize/tests/test_direct.py

```diff
@@ -116,6 +116,23 @@ def test_len_tol(self, len_tol, locally_biased):
         assert res.status == 5
         assert res.success
         assert_allclose(res.x, np.zeros((4, )))
+        message = ("The side length measure of the hyperrectangle containing "
+                   "the lowest function value found is below "
+                   f"len_tol={len_tol}")
+        assert res.message == message
+
+    @pytest.mark.parametrize("vol_tol", [1e-6, 1e-8])
```

Comment:
I'm not sure if all the parameterization over `vol_tol` is needed here, but I see the parameterization over `len_tol` above, so I suppose it's consistent.